marked-element.html

```diff
@@ -154,6 +154,13 @@
       this._attached = false;
     },
 
+    /**
+     * Unindents the markdown source that will be rendered.
+     */
+    unindent: function(text) {
+      this._unindent(text);
```

Comment:
Oh I was just being nice to breaking change people. I think I can just change the signature too?